packages/cli/lib/commands/seeder.js

```diff
@@ -0,0 +1,81 @@
+import WebSocket from 'ws'
+import { DEFAULT_PORT, RESPONSES } from '../constants.js'
+import { request } from '../utils.js'
+import { NOT_RUNNING } from './daemon.js'
+
+const ADDRESS = 'ws://localhost:' + DEFAULT_PORT
+
+seeder.usage = `
+
+  slash seeder list          - List of slashtags to be seeded while running the daemon.
+  slash seeder add [urls]    - Start seeding list of slashtags.
+  slash seeder remove [urls] - Stop seeding list of slasthags.
+`
+
+seeder.description = 'Add, remove, or list seeded slashtags'
+
+seeder.summary = 'seeder commands'
+
+export default function seeder (type, _, command) {
+  const urls = command.args.slice(1)
+  const socket = new WebSocket(ADDRESS)
+
+  socket.onopen = () => {
+    switch (type) {
+      case 'add':
+        request(socket, 'SEEDER_ADD', { urls })
```

Comment:
@Nazeh may you please say again why do you use string instead of  constant here (`'SEEDER_ADD'` instead of `REQESTS.SEEDER_ADD`) just like you do in `socket.onmessage` here, or  in `onMessge` of `packages/cli/lib/daemon/index.js 

Everything else seem to be clear now, thanks for clarifications on other comments

Reply:
The `case` deals with the value of `Request` or `Response`, and it doesn't have much typing.

On the other hand `request` function's second param needs an enum like type definition, and the easiest laziest thing I found is `keyof import('./constants.js')['RESPONSES']`, that's why I use the key.

I am sure there are ways to do [Valueof](https://stackoverflow.com/questions/49285864/is-there-a-valueof-similar-to-keyof-in-typescript)  but I don't bother be correct, since this isn't even Typescript, I just want some LSP help as easy as possible.

Reply:
@dzdidi hopefully [e70c40d](https://github.com/synonymdev/slashtags/pull/84/commits/e70c40d75a407426b150d3facd805585a62c718e) is more satisfactory